src/AppBundle/Mailjet/Message/NewsletterInvitationMessage.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace AppBundle\Mailjet\Message;
+
+use AppBundle\Entity\NewsletterInvite;
+
+final class NewsletterInvitationMessage extends MailjetMessage
+{
+    public static function createFromInvite(NewsletterInvite $invite, string $subscribeUrl): self
+    {
+        return new static(
+            $invite->getUuid(),
```

Comment:
It must be a unique UUID per message. However, the invite UUID must be used as the "batch" parameter in the message construct method. There is a dedicated optional argument for that.